plugins/woocommerce-admin/client/marketplace/components/promotions/promotions.tsx

```diff
@@ -12,37 +14,24 @@ declare global {
 	}
 }
 
-type Promotion = {
-	date_from_gmt: string;
-	date_to_gmt: string;
-	format: string;
-	pages: Page[];
-	position: string;
-	content: { [ locale: string ]: string };
-	icon?: string;
-	is_dismissible?: boolean;
-	menu_item_id?: string;
-	style?: string;
-};
-
-type Page = {
-	page: string;
-	path: string;
-	tab?: string;
-};
-
-const Promotions: () => null | JSX.Element = () => {
+const Promotions: ( {
+	promoCardOnly,
```

Comment:
This feels like a dirty hack, but it's mainly there to preserve BWC. Ideally, each ad would use the API data format (and dismiss logic) as introduced here.

Reply:
Addressed in 80078f8, much neater. Props @andfinally